test/definitions/super-agent/logs/suse155-super-agent-logs.json

```diff
@@ -0,0 +1,37 @@
+{
+    "global_tags": {
+        "owning_team": "virtuoso",
+        "Environment": "development",
+        "Department": "product",
+        "Product": "virtuoso"
+    },
+    "resources": [
+        {
+            "id": "host1",
+            "provider": "aws",
+            "type": "ec2",
+            "size": "t3.nano",
+            "ami_name": "suse-sles-15-sp5-chost-byos-v????????-hvm-ssd-x86_64"
```

Comment:
@nr-rkallempudi just want to make sure if this AMI is compatible with the ami instance mentioned in the fluent bit packages of the respective version? Also, If you think this has to be modified upon checking in the fluent bit packages repo, please modify the instance type accordingly similar to the changes done to above logs deploy config files